### DISCUSSION TITLE
android/ssl-pinning-bypass/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#80] [General]
# android/ssl-pinning-bypass/

Android SSL Pinning bypass using Frida. Frida is dynamic instrumentation toolkit for developers, reverse-engineers, and security researchers.

https://3os.org/android/ssl-pinning-bypass/

**Comment by TalonWay** [comment]
Hi,
plz add "objection root dedection bypass" functionality.

**Comment by TalonWay** [comment]
Hi,
plz add objection with multiple devices.
objection -S emulator-5556 -g com.yoyo.app explore -q // -S for devices id 
get device id from frida "frida-ls-devices"
